### PR TITLE
[16.0][MIG] base_external_dbsource_mssql

### PR DESCRIPTION
**Pull request by anddago78**
This module extends "base_external_dbsource", allowing you to connect to
foreign MSSQL databases using SQLAlchemy.

Depends: 

- [x] https://github.com/OCA/server-backend/pull/208
- [x] https://github.com/OCA/server-backend/pull/209

MT-2672 @moduon @rafaelbn @yajo @Shide  please reviews :)

### CODE REVIEW

**Review by yajo — CHANGES_REQUESTED** [review]
Dependencies are now merged, and you have conflicts. Please do the same as in https://github.com/OCA/server-backend/pull/209#pullrequestreview-1372845768. Thanks!

**Review by yajo — CHANGES_REQUESTED** [review]
Please check errors on CI.

**Comment by yajo** [comment]
Still there are errors

**Comment by anddago78** [comment]
Hi @yajo , now this does not have an error

File: base_external_dbsource_mssql/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+* Daniel Reis <dreis.pt@hotmail.com>
+* Maxime Chambreuil <maxime.chambreuil@savoirfairelinux.com>
+* Gervais Naoussi <gervaisnaoussi@gmail.com>
+* Dave Lasley <dave@laslabs.com>
+* `Tecnativa <https://www.tecnativa.com>`_:
+    * Sergio Teruel
+    * Andrea Cattalani (`Moduon <https://www.moduon.team/>`__)
```
Comment (yajo) [inline base_external_dbsource_mssql/readme/CONTRIBUTORS.rst]: Please don't put yourself inside Tecnativa.

```suggestion
* Andrea Cattalani (`Moduon <https://www.moduon.team/>`__)
```

**Review by yajo — CHANGES_REQUESTED** [review]
Just one little requirement more.

**Comment by anddago78** [comment]
Hi @yajo all comments are resolved

**Review by yajo — APPROVED** [review]

**Pull request merged**